### PR TITLE
About Page: General

### PR DESCRIPTION
**Pull request by fikrialwan**

### CODE REVIEW

**Comment by mhaidarhanif** [comment]
Thanks for the PR submission. Please rebase/merge with the `main` to get the shadcn UI config. So you can use the `lucide` icon instead of manual React SVG files.

**Review by mhaidarhanif — CHANGES_REQUESTED** [review]
Thanks for the PR submission. Please rebase/merge with the `main` to fix the conflict and get the shadcn UI setup. So you can use the `lucide` icon instead of manual React SVG files.

**Comment by fikrialwan** [comment]
I've fix the conflict and move to use "lucide" instead of use manual react svg.

File: app/components/icons/article-icon.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+export const ArticleIcon = ({
```
Comment (amrew) [inline app/components/icons/article-icon.tsx]: it should be removed, right? because you already use lucide.

Comment (fikrialwan) [reply app/components/icons/article-icon.tsx]: yeah, I've removed on this commit 6498cb1c2c1e08ee2dad5d6f1b0b36904e0b7857, maybe there something wrong with my merge. I'll fix it

Comment (fikrialwan) [reply app/components/icons/article-icon.tsx]: I've fixed it on this commit 3badc673fd3131746ff0bdc8d41b888ac6529d5c

**Review by mhaidarhanif — APPROVED** [review]
All good.

**Pull request merged**